style.css

```diff
@@ -10,46 +19,33 @@ li {
   list-style: none;
 }
 
-* {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
-
 .container {
   display: flex;
   flex-direction: column;
   flex: 1;
-  position: absolute;
-  height: 718px;
-  width: 375px;
+  position: relative;
+  width: inherit;
+  padding: auto;
 }
 
 .header {
-  width: 375px;
-  height: 92px;
   order: 1;
   background-color: white;
 }
 
 .navigation {
-  top: 44px;
   display: flex;
-  flex-direction: column;
-  height: 60px;
+  height: max-content;
   flex-wrap: wrap;
```

Comment:
- [ ] The hamburger button on your toolbar is supposed to be visible only for mobile devices! 📱 Please consider hiding it for desktop ones and adding the list of navigation links! ⛵ 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199997020-3059f639-13b1-4eae-96c7-0da35f35db96.png) | ![imageedit_11_8785617173](https://user-images.githubusercontent.com/77069011/199996732-0a3e411d-af02-4e36-8a99-bce29a107055.gif) |